docs/securing_fledge.rst

```diff
@@ -142,11 +148,17 @@ The user management pages allows
   - Deleting users.
   - Resetting user passwords.
   - Changing the role of a user.
+  - Changing the details of a user
+
+Fledge currently supports four roles for users:
+
+  - **Administrator**: a user with admin role is able to fully configure Fledge, view the data read by the Fledge instance  and also manage Fledge users
 
-Fledge currently supports two roles for users,
+  - **Editor**: a user with this role is able to configure Fledge and view the data read by Fledge. The user can not manage other users or add new users.
 
-  - **admin**: a user with admin role is able to fully configure Fledge and also manage Fledge users
-  - **user**: a user with this role is able to configure Fledge but can not manage users
+  - **Viewer**: a user that can only view the configuration of the Fledge instance. The user has no ability to modify the Fledge instance in any way.
+
+  - **Data Viewer**: a user that can only view the data in Fledge and not the configuration of Fledge itself. The user has no ability to modify the Fledge instance in any way.
```

Comment:
> view the data

We should tell to users - From GUI tabs like `Assets & Readings, Dashboards etc.` can be viewed.

Reply:
I would rather not lit specific items that are or are not available as it will need constant changing as we add features.

Reply:
Sure